nodemon.json

```diff
@@ -3,6 +3,6 @@
     "server",
     "client"
   ],
-  "ext": "js, ejs",
+  "ext": "js",
```

Comment:
@gnorbsl no need to restart the server on ejs file changes

Reply:
Ah makes sense, I somehow thought that pages get rendered once on server start :D

Reply:
nope, only on load in browser, we don't use webpack yet for hot reloading